### DISCUSSION TITLE
"Unable to find package" when attempting to add NuGet package

### DISCUSSION THREAD

**Discussion by glen-84** [community/community#40767] [Actions]
### Select Topic Area

Product Feedback

### Body

How do I install [this package](https://github.com/glen-84/analyzer-config-test/packages/1741697?version=1.0.0)?

When I run `dotnet add src/Application/ package Glen84.AnalyzerConfig.Applications --version 1.0.0`, I get:

```
  Determining projects to restore...
  Writing /tmp/tmpqfgBim.tmp
info : X.509 certificate chain validation will use the fallback certificate bundle at '/home/glen/.dotnet/sdk/6.0.402/trustedroots/codesignctl.pem'.
info : Adding PackageReference for package 'Glen84.AnalyzerConfig.Applications' into project '/home/glen/development-environment/projects/api-next/src/Application/Application.csproj'.
info : Restoring packages for /home/glen/development-environment/projects/api-next/src/Application/Application.csproj...
info :   GET https://nuget.pkg.github.com/glen-84/download/glen84.analyzerconfig.applications/index.json
info :   NotFound https://nuget.pkg.github.com/glen-84/download/glen84.analyzerconfig.applications/index.json 1251ms
error: NU1101: Unable to find package Glen84.AnalyzerConfig.Applications. No packages exist with this id in source(s): github
error: Package 'Glen84.AnalyzerConfig.Applications' is incompatible with 'all' frameworks in project '/home/glen/development-environment/projects/api-next/src/Application/Application.csproj'.
```

`src/Application/nuget.config`:

```xml
<?xml version="1.0" encoding="utf-8" ?>
<configuration>
    <packageSources>
        <clear />
        <add key="github" value="https://nuget.pkg.github.com/glen-84/index.json" />
    </packageSources>
    <packageSourceCredentials>
        <github>
            <add key="Username" value="glen-84" />
            <add key="ClearTextPassword" value="(my token here)" />
        </github>
    </packageSourceCredentials>
</configuration>
```

Am I doing something wrong?

**Comment by ossanna16** [comment]
Hi there @glen-84 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by glen-84** [comment]
I just tried again now and it worked. 🤷‍♂️
